core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/SQLPlanParser.scala

```diff
@@ -297,10 +297,12 @@ object ExecInfo {
     // check is the node has a dataset operations and if so change to not supported
     val rddCheckRes = RDDCheckHelper.isDatasetOrRDDPlan(nodeName, node.desc)
     val ds = dataSet || rddCheckRes.isRDD
+    val containsPythonUDF = ExecHelper.isPythonUDF(node)
+    val finalSupported = isSupported || containsPythonUDF
 
     // if the expression is RDD because of the node name, then we do not want to add the
     // unsupportedExpressions because it becomes bogus.
-    val finalUnsupportedExpr = if (rddCheckRes.nodeDescRDD) {
+    val finalUnsupportedExpr = if (rddCheckRes.nodeDescRDD || containsPythonUDF) {
```

Comment:
I don't think this is correct.
The code comment just above that change is explaining that we avoid parsing the node content when it is `rddCheckRes.nodeDescRDD` . But the new code tries to check if it is pythonUDF anyway which could lead to bogus results.

Reply:
Sorry for the confusion. I will address this in the next iteration.